core/trino-main/src/main/java/io/trino/connector/system/KillQueryProcedure.java

```diff
@@ -60,6 +61,9 @@ public KillQueryProcedure(Optional<DispatchManager> dispatchManager, AccessContr
     @UsedByGeneratedCode
     public void killQuery(String queryId, String message, ConnectorSession session)
     {
+        checkProcedureArgument(queryId != null, "query_id cannot be null");
+        checkProcedureArgument(message != null, "message cannot be null");
```

Comment:
Looks like `message` can be null: https://github.com/trinodb/trino/blob/master/core/trino-main/src/main/java/io/trino/connector/system/KillQueryProcedure.java#L104

Reply:
Aah, I see. Then should we make message an optional field?

https://github.com/trinodb/trino/blob/91b49be0736be254bbc360cbe70899deac25720c/core/trino-main/src/main/java/io/trino/connector/system/KillQueryProcedure.java#L96

Reply:
Yeah, I don't see why it shouldn't be optional.